code/src/main/java/com/google/sps/data/PositiveUser.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+@Entity
+public class PositiveUser {
+
+  @Id String userId;
+  private int emailsSent = 0;
+  private long firstLoginUnixTimeSeconds;
+  private long lastLoginUnixTimeSeconds;
+
+  private PositiveUser() {}
+  
+  public PositiveUser(String userId) {
+    this.userId = userId;
+    firstLoginUnixTimeSeconds = lastLoginUnixTimeSeconds = Instant.now().getEpochSecond();
+  }
+
+  public void setLastLogin() {
```

Comment:
did grady say public methods need comments?

Reply:
Not sure

Reply:
But yes I can add comments to every method if yall want

Reply:
We don't need them on pure getters or setters, but yes, all other methods should have  javadoc.